### PR TITLE
fix(HMS-1567): insights tags support

### PR DESCRIPTION
**Pull request by lzap**
This adds a temporary solution to the inventory registration problem - it will take us more time to fully implement host pre-registration. To showcase a reasonable workflow to users, tags allows to present both IPv4 and FQDN of AWS public info. This is downloaded from the metadata interface and then uploaded as tags by insights client.

Note, when system is shutdown it will get a different credentials. This will not reflect such a change, it is an easy hack that will be removed as soon as we add a pre-registration feature.

### CODE REVIEW

**Comment by lzap** [comment]
Do not merge I am also adding Azure.

**Review by adiabramovitch — APPROVED** [review]
Looks great to me, haven't check that though. 
Just verifying:
After merging this PR instances launched using prod beta (or all environments?) are supposed to be registered to insights?

File: internal/clients/instance_params.go
Comment (lzap) [reply internal/clients/instance_params.go:24]: Yes, GCP does not support cloud-init natively, you need to install [google cloud platform tools](https://cloud.google.com/compute/docs/manage-os#rhelcentosrocky-linux). Image builder should put it there by default when you create a GCP image.

That is also the reason why the template for GCP is not cloud init (YAML) but a shell script.

**Comment by lzap** [comment]
> After merging this PR instances launched using prod beta (or all environments?) are supposed to be registered to insights?

With this patch, when a system is registered to insights either via activation key (automatically) or manually via `rhc register`, the hosts will have the IP information in the insights UI. You will find it under Tags section.

I tested AWS and Azure myself, not GCP. But please test it all there is not much time for fixing this this patch needs to be correct from the day one :-)

**Comment by lzap** [comment]
I added `mkdir` in case insights client is not installed during firstboot. Try now @adiabramovitch

**Comment by lzap** [comment]
I tested AWS and GCP.

Do not merge until someone tests Azure, I might not be able to finish this today.

**Comment by lzap** [comment]
Going to re-test Azure, I think it was a success but I will check again.

**Comment by lzap** [comment]
And Azure does not work:

```
# /usr/bin/curl -sH "Metadata:true" --connect-timeout 5 http://169.254.169.254/metadata/loadbalancer?api-version=2020-10-01
{ "error": "No load balancer metadata is found. Please check if your VM is using any non-basic SKU load balancer and retry later." }
```

File: internal/db/seeds/dev_small.sql
Comment (lzap) [inline internal/db/seeds/dev_small.sql:25]: I am adding my RSA key to the seed, also I found that fingerprint of the previous key (ID=1) was incorrectly set. This fixes it. I need this because Azure does not support elyptic curves keys.

File: internal/clients/instance_params.go
Comment (adiabramovitch) [reply internal/clients/instance_params.go:24]: I meant you have a typo :D 
you wrote CGP

**Comment by ezr-ondrej** [comment]
So lets get that in? :)
Looks green and tested

**Review by adiabramovitch — COMMENTED** [review]
other than the typo I am good here

**Comment by lzap** [comment]
Oh fixed the type, did not see it :-) Rebased.

**Pull request merged**
